app/src/main/java/com/fs_sournary/weather/utils/Constant.java

```diff
@@ -0,0 +1,13 @@
+package com.fs_sournary.weather.utils;
+
+/**
+ * Created by fs-sournary.
+ * Date on 11/17/17.
+ * Description:
+ */
+
+public class Constant {
+
+    public static String CURRENT_FRAGMENT_TAG = "current_fragment";
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName